### PR TITLE
[ProductVideo] Validating the Product Video Url

### PR DESCRIPTION
**Pull request by eduard13**
### Description (*)
This PR adds validation for getting the video information, after clicking on `Get Video Information` button.

### Fixed Issues (if relevant)

1. magento/magento2#25088: Get Video Information button doesn't do anything on clicked

### Manual testing scenarios (*)

1. Navigate to new product page
2. Scroll to Images And Videos
3. Click on Add Video
4. Click on Get Video Information

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/ProductVideo/Test/Mftf/ActionGroup/AdminAssertVideoValidationErrorActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertVideoValidationErrorActionGroup">
```
Comment (rogyar) [inline app/code/Magento/ProductVideo/Test/Mftf/ActionGroup/AdminAssertVideoValidationErrorActionGroup.xml]: Please use Assert[Area]Desctiption naming format for action groups (https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html 11.3.5)

File: app/code/Magento/ProductVideo/Test/Mftf/Test/AdminValidateUrlOnGetVideoInformationTest.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        <actionGroup ref="AdminOpenProductIndexPageActionGroup" stepKey="navigateToProductIndexPage"/>
+        <actionGroup ref="goToCreateProductPage" stepKey="goToCreateProduct">
+            <argument name="product" value="SimpleProduct"/>
+        </actionGroup>
+        <actionGroup ref="AdminOpenProductVideoModalActionGroup" stepKey="openAddProductVideoModal"/>
+        <actionGroup ref="AdminGetVideoInformationActionGroup" stepKey="clickOnGetVideoInformation"/>
+        <actionGroup ref="AdminAssertVideoValidationErrorActionGroup" stepKey="seeUrlValidationMessage">
+            <argument name="inputName" value="video_url"/>
+        </actionGroup>
+        <fillField selector="{{AdminProductNewVideoSection.videoUrlTextField}}"
```
Comment (rogyar) [inline app/code/Magento/ProductVideo/Test/Mftf/Test/AdminValidateUrlOnGetVideoInformationTest.xml]: Please, create a separate action group (like AdminFillProductVideoUrlActionGroup) for filling this field and pass the value as argument.

**Review by rogyar — CHANGES_REQUESTED** [review]

**Comment by eduard13** [comment]
Hi @rogyar, thank you for reviewing it, the requested changes were adjusted.

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-6134](https://jira.corp.magento.com/browse/ENGCOM-6134) has been created to process this Pull Request
:eight_spoked_asterisk: @rogyar, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**